components/Article/Embeds/Embeds.js

```diff
@@ -0,0 +1,136 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Block from '../../Block';
+
+export const FacebookEmbed = ({
```

Comment:
Acho melhor separarmos isso aqui em arquivos. Fica mais organizado e evita quando crescer termos um arquivo de 200 linhas com embeds. Podemos até separar em folders amp e web caso seja necessário...

Reply:
To com uma dúvida quando aos embeds do non-amp. Devemos instalar no prensa ou no projeto?